private-templates-service/adaptivecards-templating-client/typescript/api.ts

```diff
@@ -677,6 +759,63 @@ export class TemplateApi {
             });
         });
     }
+    /**
+     * Template preview for shareable link
+     * @summary Get template preview
+     * @param templateId ID of template to return
+     * @param {*} [options] Override http request options.
+     */
+    public templatePreviewById (templateId: string, options: any = {}) : Promise<{ response: http.IncomingMessage; body: TemplatePreviewResponse;  }> {
+        const localVarPath = this.basePath + '/template/{templateId}/preview'
```

Comment:
`const localVarPath = '${this.basePath}/template/${encodeURIComponent(String(templateId)))}/preview'`